Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN pip install --upgrade --no-cache-dir black
+
```

Comment:
an additional improvement to execution here is to run black once such that it puts the grammar cache into the image, from my experiments this improves execution of black by about ~50ms per run

one way that I did this was by changing the install line to:

```dockerfile
RUN : \
    && pip install --upgrade --no-cache-dir black \
    # have black run against a file so its cache and pyc files are populated
    && touch /tmp/t.py && black /tmp/t.py && rm /tmp/t.py
```